### DISCUSSION TITLE
🎉 Choco is now available as a python package 🚀

### DISCUSSION THREAD

**Discussion by cprudhom** [chocoteam/choco-solver#942] [Announcements]
Thanks to the great work of @dimitri-justeau we are very pleased to announce that a Python bindings for`choco-solver` is now available !

That's beta version but almost all constraints are supported. By the way, we are looking for beta-testers 😜 

You will find more details on the `pychoco` [GitHub repo](https://github.com/chocoteam/pychoco/releases/tag/pychoco-0.1.1).

**Comment by dimitri-justeau** [comment]
Choco lovers, please see https://github.com/chocoteam/pychoco/discussions/3 for some news and perspectives about pychoco :-)
